kafka-starter-testing/src/main/java/org/galatea/kafka/starter/testing/TopologyTester.java

```diff
@@ -56,12 +54,10 @@
   private final Map<String, TopicConfig<?, ?>> outputTopicConfig = new HashMap<>();
   private final Map<String, TopicConfig<?, ?>> storeConfig = new HashMap<>();
   private final Set<Class<?>> beanClasses = new HashSet<>();
-  private final Set<Class<?>> avroClasses = new HashSet<>();
 
   @Getter
   private final ConversionUtil typeConversionUtil = new ConversionUtil();
-  @Getter
-  private final AvroMessageUtil avroMessageUtil = AvroMessageUtil.defaultUtil();
+  private final Map<Class<?>, RecordPostProcessor<?>> postProcessors = new HashMap<>();
```

Comment:
Use `Maps.newHashMap()` to be in with the cool kids 😎

Reply:
ehh, I see that as a helper method that's no longer useful, with the introduction of <> in Java8. The only difference is that maps.newHashMap() requires additional library.

Reply:
Ahh ok, I just find it a bit neater -- personal preference dw